taiga/webhooks/tasks.py

```diff
@@ -95,10 +95,16 @@ def _send_request(webhook_id, url, key, data):
 
     with requests.Session() as session:
         try:
-            response = session.send(prepared_request)
+            response = session.send(prepared_request, allow_redirects=settings.WEBHOOKS_ALLOW_REDIRECTS)
+
+            if not settings.WEBHOOKS_ALLOW_REDIRECTS and response.status_code in [301, 302, 303, 307, 308]:
+                raise RequestException("Redirects are not allowed")
+
         except RequestException as e:
             # Error sending the webhook
-            webhook_log = WebhookLog.objects.create(webhook_id=webhook_id, url=url, status=0,
+            webhook_log = WebhookLog.objects.create(webhook_id=webhook_id,
+                                                    url=url,
+                                                    status=response.status_code or 0,
```

Comment:
I guess the exception is rendered in the Webhooks trials in Taiga, but why not to propagate the status? For instance, in case of a 3XX status it would provide clues that it may be a settings problem (with the new `WEBHOOKS_ALLOW_REDIRECTS`).

Does it reveal unintended information?

Reply:
That is the change I propose. Before, the state was always 0 `status=0`. Now, the value of the status will be that of the request or 0 if it does not have one `status=response.status_code or 0`.